packages/calcite-components/src/components/split-button/split-button.scss

```diff
@@ -14,28 +172,134 @@
   @apply w-full;
 }
 
+//--calcite-internal-split-button-loader-color: var(--calcite-color-inverse);
```

Comment:
Commented out line should be removed. Applies to other commented-out lines.

If any commented-out line is waiting on a decision or question, let's add context and keep the comment.

Reply:
thanks for the catch